### DISCUSSION TITLE
Version information for 2.2 shows 2.2-beta

### DISCUSSION THREAD

**Discussion by anandkaranam** [linux-test-project/lcov#336] [Q&A]
Is it correct and expected that the version information for lcov version 2.2 is shown as 
2.2-beta or is it something to be fixed?

**Comment by henry2cox** [comment]
Not correct/my error - but also not something worth to fix, I think.

You can see from the labels that there is no difference between the 2.2-beta and 2.2 release labels (because nobody noticed or reported any problems in the beta period)...and I just reused the same tarball without thinking about embedded labels.

Subsequently, a number of issues have been reported (and fixed) and MC/DC support has been added - so I think it is more useful to just make a 2.3 release (beta first, then numbered) rather than fixing the 2.2 artifacts.

Perhaps someone can make a good argument why the above should be done differently.

**Reply by anandkaranam** [reply]
agree that it is not something worth to fix, but to consider the input for 2.3 release.
